src/details/BookDetails.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import MarketplaceLink from './MarketplaceLink'
+import StarRating from './StarRating'
+import BookYearsReferenced from './BookYearsReferenced'
+
+class BookDetails extends Component {
+
+    createMarkup(book) {
+        return {__html: book.summary};
+    }
+
+    render() {
+        let book = this.props.book;
```

Comment:
is there a reason to use "let" instead of "const" here? I'm actually not sure how this works when it's reacting to the passed down content of the book. "let" could be correct... I'm just not sure.